app/services/exchange_rates/monthly_exchange_rates_service.rb

```diff
@@ -22,6 +23,10 @@ def notify
         SlackNotifierService.call(message)
       end
 
+      def email_files_to_hmrc
+        ExchangeRates::Mailer.monthly_files
```

Comment:
Not sure if this is pattern used elsewhere but thought this might be something like `Mailer::MonthlyExchangeRates.call` then we move all the mailer configuration into its own folder so future developers can easily replicate how we use mailing and it wont change per business area and logic can be shared?

Reply:
So `mailer` folder is moved up a level even?

Reply:
the way i'm doing is how it's done in the current app code in tariff_sync mailer